third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -183,12 +183,15 @@ std::optional<xla::OpSharding> PjRtComputationClient::GetDataSharding(
 
 std::vector<ComputationClient::DataPtr> PjRtComputationClient::TransferToServer(
     absl::Span<const TensorSource> tensors) {
-  metrics::TimedSection timed(TransferToServerMetric());
-  tsl::profiler::TraceMe activity("PjRtComputationClient::TransferToServer",
-                                  tsl::profiler::TraceMeLevel::kInfo);
+  auto timed = std::make_shared<metrics::TimedSection>(TransferToServerMetric());
+  tensorflow::profiler::TraceMe activity(
```

Comment:
This should stay `tsl::profiler`, right?

Reply:
In ExecuteComputation, I see that its tensorflow::profiler. Whats the difference b/w tensorflow::profiler and tsl::profiler?